discord/webhook.py

```diff
@@ -45,10 +45,100 @@
     'RequestsWebhookAdapter',
     'Webhook',
     'WebhookMessage',
+    'PartialWebhookChannel',
+    'PartialWebhookGuild'
 )
 
 log = logging.getLogger(__name__)
 
+class PartialWebhookChannel(Hashable):
+    """Partial channel for webhooks for when they are channel following webhooks for news channels.
```

Comment:
This docstring doesn't match the rest of the codebase.

```suggestion
    """Represents a partial channel for webhooks.

    These are typically given for channel follower webhooks.
```